src/hw/connectApp.js

```diff
@@ -7,14 +7,17 @@ import {
   FirmwareOrAppUpdateRequired,
   UserRefusedOnDevice,
   BtcUnmatchedApp,
+  UpdateYourApp,
```

Comment:
The error was always in the "errors" package or does the PR need a package.json and/or package.lock bump ?

Reply:
It's already there yeah